jumeg/decompose/ica.py

```diff
@@ -794,3 +794,27 @@ def ica2data_single_components(sources, ica, pca, picks=None):
     # x_mean_comp[:, icomp] = data.mean(axis=1)
 
     return data
+
+
+#######################################################
+#                                                     #
+# apply ICA based on filtered data to unfiltered raw  #
+#                                                     #
+#######################################################
+def ica_apply_unfiltered(raw_unfilt, ica_filt, picks):
+
+    data = raw_unfilt.get_data(picks, None, None, None)
```

Comment:
It would be useful to expose the start, stop parameters here for use when applying ICA on chops.

Reply:
@pravsripad Thanks, this can easily be added to the function